website/docs: add release notes for 2026.8 (#24412)
goauthentik/authentik: website/docs/releases/2026/v2026.8.md

## Highlights

- **Privileged access management (PAM)**: :ak-enterprise Users can request access to applications or specific application entitlements. Designated approvers can approve or deny requests and set when access expires.
- **User switching**: Users can keep multiple accounts signed in within the same browser and switch between them from the User interface.

> - **User expiration and offboarding**: :ak-enterprise Automate account expiration after inactivity or schedule users for deactivation or deletion, with optional session and token revocation.

Reviewer: User expiration is not yet included in this release, just scheduled offboarding
 
```suggestion
- **User offboarding**: :ak-enterprise Schedule users for deactivation or deletion, with optional session and token revocation.
```

Administrators define request rules that control who can request and approve access. Rules support approval by individual users, groups, or policies, minimum reviewer counts, request and grant expiration limits, reviewer notifications, and custom flows that collect request details. Reviewers can approve or deny pending requests and revoke active grants. Each action is recorded in the event log.

For more details, refer to the [privileged access management documentation](TODO).

### User switching

Users can now keep multiple authentik accounts signed in within the same browser and switch between them from the account menu in the User interface. They can also add another account without signing out of their current accounts.

Administrators enable the feature by selecting a **User switch flow** for the brand. Because this is a normal authentik flow, policies can control whether a switch requires a password, MFA, or a shorter verification process. Successful switches are recorded in the event log. For configuration details, refer to the [user switching documentation](../../users-sources/user/user-switching.md).

### User expiration and scheduled offboarding :ak-enterprise

> User accounts can now be automatically deactivated or deleted after a configured period of inactivity. Administrators can also schedule a user for deactivation or deletion at a specific date and time, with the option to revoke the user's sessions and tokens.

Reviewer: ```suggestion
### Scheduled user offboarding :ak-enterprise

Administrators can schedule a user for deactivation or deletion at a specific date and time, with the option to revoke the user's sessions and tokens.
```

### Object attributes

Administrators can now define custom text, number, and Boolean fields under **Directory** > **Object attributes**. Fields can be required, unique, or validated against a regular expression.

These fields appear when editing users, groups, application entitlements, and device access groups, and the same validation applies when objects are managed through the API.

authentik also includes definitions for common identity, contact, address, Unix, and employee attributes. These definitions are disabled by default and can be enabled as needed.

### Self-hosted event maps :ak-enterprise

The events map now uses a basemap bundled with authentik and does not make external requests by default, making it suitable for air-gapped and restricted-network deployments. Nearby events are grouped together, with columns showing the event volume and the types of events recorded in each area.

Users can select an area to filter the event list, and each brand can use the bundled map or configure a self-hosted map under **Map tiles**. For more details, refer to the [events documentation](../../sys-mgmt/events/index.md).

> ### OAuth 2.0 and OpenID Connect

Reviewer 2: ```suggestion
### OAuth 2.0 and OpenID Connect

#### OpenID Certification

As of 2026.8, authentik is officially **OpenID Certified™** by the OpenID Foundation — for both [OpenID Provider profiles](https://openid.net/certification/certified-openid-providers-profiles/) (Basic, Implicit, Hybrid, Config, and Form Post OP) and [logout profiles](https://openid.net/certification/certified-openid-providers-for-logout-profiles/) (RP-Initiated, Front-Channel, and Back-Channel).
```

- [Icinga Web 2](https://integrations.goauthentik.io/monitoring/icinga/) (Thanks to @nicedevil007!)
- [IIS](https://integrations.goauthentik.io/infrastructure/iis/) (Thanks to @dominic-r!)
- [Infisical](https://integrations.goauthentik.io/security/infisical/) (Thanks to @dominic-r!)
- [Kavita](https://integrations.goauthentik.io/media/kavita/) (Thanks to @PeshekDotDev!)
- [mailcow Logs Viewer](https://integrations.goauthentik.io/chat-communication-collaboration/mailcow-logs-viewer/) (Thanks to @nicedevil007!)
- [Memos](https://integrations.goauthentik.io/chat-communication-collaboration/memos/) (Thanks to @dominic-r!)
- [n8n](https://integrations.goauthentik.io/development/n8n/) (Thanks to @dominic-r!)
- [Nexterm](https://integrations.goauthentik.io/infrastructure/nexterm/) (Thanks to @Nerdy-Technician!)
- [NocoDB](https://integrations.goauthentik.io/development/nocodb/) (Thanks to @dominic-r!)
- [Notion](https://integrations.goauthentik.io/documentation/notion/) (Thanks to @dominic-r!)
- [Omnissa Workspace ONE Access](https://integrations.goauthentik.io/device-management/omnissa-workspace-one-access/) (Thanks to @nicedevil007!)

> - [OpenCloud](https://integrations.goauthentik.io/chat-communication-collaboration/opencloud/) (Thanks to @PeshekDotDev!)

Reviewer 2: We ended up having a conversation and deciding that any integrations done by the team (me, dewi, dominic, etc), that we don't link to the github. We want outside contributors to stand out more for this

Reviewer 2: @PeshekDotDev, @dominic-r, @dewi-tik

Reviewer 2: Didn't mean to tag everyone, just wanted to put the names for ken lol

- [Personio](https://integrations.goauthentik.io/platforms/personio/) (Thanks to @nicedevil007!)
- [Plane](https://integrations.goauthentik.io/chat-communication-collaboration/plane/) (Thanks to @dominic-r!)
- [Productive](https://integrations.goauthentik.io/platforms/productive/) (Thanks to @dominic-r!)
- [SnapOtter](https://integrations.goauthentik.io/media/snapotter/) (Thanks to @dominic-r!)
- [Stirling PDF](https://integrations.goauthentik.io/documentation/stirling-pdf/) (Thanks to @dominic-r!)
- [ToolJet](https://integrations.goauthentik.io/platforms/tooljet/) (Thanks to @dominic-r!)
- [Veeam Backup & Replication](https://integrations.goauthentik.io/infrastructure/veeam-backup-replication/) (Thanks to @nicedevil007!)

> ## New source guides

Reviewer 2: Love this as a separate section, very smart

### Self-hosted event maps :ak-enterprise

The events map now uses a basemap bundled with authentik and does not make external requests by default, making it suitable for air-gapped and restricted-network deployments. Nearby events are grouped together, with columns showing the event volume and the types of events recorded in each area.

Users can select an area to filter the event list, and each brand can use the bundled map or configure a self-hosted map under **Map tiles**. For more details, refer to the [events documentation](../../sys-mgmt/events/index.md).

### OAuth 2.0 and OpenID Connect

#### OpenID Certification

As of 2026.8, authentik is officially **OpenID Certified™** by the OpenID Foundation — for both [OpenID Provider profiles](https://openid.net/certification/certified-openid-providers-profiles/) (Basic, Implicit, Hybrid, Config, and Form Post OP) and [logout profiles](https://openid.net/certification/certified-openid-providers-for-logout-profiles/) (RP-Initiated, Front-Channel, and Back-Channel).

> 

Reviewer 3: Congratulation Authentik!

Reviewer 4: Thanks! All the credit goes @PeshekDotDev who did the hard part!

- [NocoDB](https://integrations.goauthentik.io/development/nocodb/)
- [Notion](https://integrations.goauthentik.io/documentation/notion/)
- [Omnissa Workspace ONE Access](https://integrations.goauthentik.io/device-management/omnissa-workspace-one-access/) (Thanks to @nicedevil007!)
- [OpenCloud](https://integrations.goauthentik.io/chat-communication-collaboration/opencloud/)
- [Personio](https://integrations.goauthentik.io/platforms/personio/) (Thanks to @nicedevil007!)
- [Plane](https://integrations.goauthentik.io/chat-communication-collaboration/plane/)
- [Productive](https://integrations.goauthentik.io/platforms/productive/)
- [SnapOtter](https://integrations.goauthentik.io/media/snapotter/)
- [Stirling PDF](https://integrations.goauthentik.io/documentation/stirling-pdf/)
- [ToolJet](https://integrations.goauthentik.io/platforms/tooljet/)
- [Veeam Backup & Replication](https://integrations.goauthentik.io/infrastructure/veeam-backup-replication/) (Thanks to @nicedevil007!)

## New source guides

The following source guides were recently added. A big thanks to our contributors!

- [GitLab](../../users-sources/sources/social-logins/gitlab/index.mdx) (Thanks to @4www!)

> - [JumpCloud](../../users-sources/sources/social-logins/jumpcloud/index.mdx) (Thanks to @PeshekDotDev!)

Reviewer 2: ```suggestion
- [JumpCloud](../../users-sources/sources/social-logins/jumpcloud/index.mdx)
```
